src/main/java/net/minecraftforge/registries/ForgeRegistry.java

```diff
@@ -167,12 +168,13 @@ public ResourceLocation getRegistryName()
         return superType;
     }
 
-    @Nullable
-    public String getTagFolder()
+    @Override
+    public boolean supportsTags()
     {
-        return tagFolder;
+        return this.builder.getHasWrapper();
```

Comment:
I feel like this potentially should just be stored in a local variable the same as everything else that references the builder.